src/invoke.rs

```diff
@@ -1,8 +1,17 @@
-use std::{fmt::Debug, fs, io};
+use std::{
+    fmt::Debug,
+    fs::{self, File},
+    io,
+    io::BufRead,
+    io::Write,
+    rc::Rc,
+};
 
 use clap::Parser;
 use stellar_contract_env_host::{
-    xdr::{Error as XdrError, ScVal, ScVec},
+    im_rc::OrdMap,
```

Comment:
It's fine if you want to use the `im_rc::OrdMap`, but in the CLI it is probably unnecessary to do so. You could use the stdlib's [`BTreeMap`](https://doc.rust-lang.org/std/collections/struct.BTreeMap.html) which retains order and doesn't require depending on any interesting dependencies.

Reply:
`storage.map`, which we pass to `snapshot::commit`, is an `OrdMap`, so it's seems simpler to use that everywhere instead of also including `BTreeMap`. What do you think?